integrations/axum/src/lib.rs

```diff
@@ -180,7 +180,7 @@ pub async fn generate_request_and_parts(
 ///
 /// This can then be set up at an appropriate route in your application:
 ///
-/// ```
+/// ```rust,ignore
```

Comment:
I needed to add these to make tests pass. Some tests seemingly unrelated to what I changed still seem to be failing.